packages/ui/src/app/pages/Settings/SettingsNotificationsTab.stories.tsx

```diff
@@ -0,0 +1,186 @@
+import { expect } from '@storybook/jest'
+import { Meta, StoryContext, StoryObj } from '@storybook/react'
+import { userEvent, waitFor, within } from '@storybook/testing-library'
+import { FC } from 'react'
+
+import {
+  GetBackendMeDocument,
+  GetBackendMemberExistsDocument,
+  UpdateBackendMemberDocument,
+} from '@/memberships/queries/__generated__/backend.generated'
+import { member } from '@/mocks/data/members'
+import { getButtonByText, withinModal } from '@/mocks/helpers'
+import { MocksParameters } from '@/mocks/providers'
+
+import { SettingsNotificationsTab } from './SettingsNotificationsTab'
+
+type Args = {
+  isRegistered: boolean
+  isEmailConfirmed: boolean
+  onMemberUpdate: CallableFunction
+}
+type Story = StoryObj<FC<Args>>
+
+const alice = member('alice')
+const email = 'test@example.com'
+
+export default {
+  title: 'Pages/Settings/Notifications',
+  component: SettingsNotificationsTab,
+
+  argTypes: {
+    isRegistered: { control: { type: 'boolean' } },
+    isEmailConfirmed: { control: { type: 'boolean' } },
+    onMemberUpdate: { action: 'MemberUpdate' },
+  },
+
+  args: {
+    isRegistered: true,
+    isEmailConfirmed: true,
+  },
+
+  parameters: {
+    isLoggedIn: true,
+    balance: 100,
+
+    mocks: ({ args, parameters }: StoryContext<Args>): MocksParameters => {
+      return {
+        accounts: parameters.isLoggedIn
+          ? { active: { member: alice, balances: parameters.balance } }
+          : { list: [{ member: alice }] },
+
+        gql: {
+          queries: [
+            {
+              query: GetBackendMemberExistsDocument,
+              data: { memberExist: args.isRegistered },
```

Comment:
Just to easily check these states:
```suggestion
              resolver: () => {
                data: { memberExist: args.isRegistered },
                isLoading: args.activeMemberExistBackendLoading,
                error: args.activeMemberExistBackendError,
              }
```

Reply:
👍